src/app/posts/posts.component.html

```diff
@@ -1,26 +1,28 @@
 <div *ngFor="let p of posts" class="post">
```

Comment:
let POST of posts
Please use correct naming for variables, it is important part of development!!!